### PR TITLE
Update to the UCT overview page

### PR DESCRIPTION
**Pull request by sgonzalezmangana**
## Purpose of this pull request

This pull request (PR) makes some editorial changes.

## Affected pages

- https://experienceleague.adobe.com/docs/commerce-operations/upgrade-guide/overview.html

### CODE REVIEW

**Comment by sgonzalezmangana** [comment]
@sivaschenko @danidelcar I'm reviewing UCT documentation and updating the content that is not too technical. Please have a look to these minor changes

**Comment by jeff-matthews** [comment]
@sgonzalezmangana, you'll need to sign the Adobe CLA before we can finish processing this contribution.

@sivaschenko, I sent you an invitation to join the AdobeDocs org so that we can assign you as a reviewer.

File: help/upgrade/upgrade-compatibility-tool/overview.md [hunk 1]
```diff
@@ -19,12 +19,12 @@ Refer to the [Install](../upgrade-compatibility-tool/install.md) for the first s
...
 
->[!NOTE]
->
->We want to hear your feedback, issues, and suggestions to help us improve the tool.
-
-Do you have an idea how to improve our compatibility analysis? Check the [Contribute](https://devdocs.magento.com/guides/v2.4/coding-standards/contributing.html) page for more information.
+The Upgrade Compatibility Tool uses rules defined within our [Coding Standards](https://devdocs.magento.com/guides/v2.4/coding-standards/bk-coding-standards.html), to ensure your project is following Adobe Commerce best practices, and at the same time allows you to improve and extend the Upgrade Compatibility Tool compatibility analysis by contributing to this project. Check the [Contribute](https://devdocs.magento.com/guides/v2.4/coding-standards/contributing.html)  page for more information.
 
-See the [Resources](https://devdocs.magento.com/community/resources/resources.html) page for more information.
+## Resources
```
Comment (danidelcar) [inline help/upgrade/upgrade-compatibility-tool/overview.md:28]: Missing a couple of spaces in between title and lists:

```
## Resources

We have developed other resources to help you understand Adobe Commerce upgrades, these are links to key resources:

- [Upgrade guide](https://experienceleague.adobe.com/docs/commerce-operations/assets/adobe-commerce-2-4-upgrade-guide.pdf)
- [Upcoming releases](https://devdocs.magento.com/release/)
- [Community Resources](https://devdocs.magento.com/community/resources/resources.html) page for more information.

```

File: help/upgrade/upgrade-compatibility-tool/overview.md [hunk 2]
```diff
@@ -19,12 +19,12 @@ Refer to the [Install](../upgrade-compatibility-tool/install.md) for the first s
...
+To connect with the [!DNL Upgrade Compatibility Tool] team, contact us on the Engineering Slack channel [[!DNL Upgrade Compatibility Tool]](https://magentocommeng.slack.com/archives/C019Y143U9F). We want to hear your feedback, issues, and suggestions to help us improve the tool.
 
->[!NOTE]
->
->We want to hear your feedback, issues, and suggestions to help us improve the tool.
-
-Do you have an idea how to improve our compatibility analysis? Check the [Contribute](https://devdocs.magento.com/guides/v2.4/coding-standards/contributing.html) page for more information.
+The Upgrade Compatibility Tool uses rules defined within our [Coding Standards](https://devdocs.magento.com/guides/v2.4/coding-standards/bk-coding-standards.html), to ensure your project is following Adobe Commerce best practices, and at the same time allows you to improve and extend the Upgrade Compatibility Tool compatibility analysis by contributing to this project. Check the [Contribute](https://devdocs.magento.com/guides/v2.4/coding-standards/contributing.html)  page for more information.
```
Comment (jeff-matthews) [inline help/upgrade/upgrade-compatibility-tool/overview.md]:
```suggestion
The [!DNL Upgrade Compatibility Tool] uses rules defined within our [Coding Standards](https://devdocs.magento.com/guides/v2.4/coding-standards/bk-coding-standards.html) to ensure that your project is following Adobe Commerce best practices and to help you improve and extend the [!DNL Upgrade Compatibility Tool]. Check the [Contributing](https://devdocs.magento.com/guides/v2.4/coding-standards/contributing.html) page for more information.
```

Comment (danidelcar) [inline help/upgrade/upgrade-compatibility-tool/overview.md]: Don't forget to include the tag, so name is not translated nor localized: [!DNL Upgrade Compatibility Tool].
So:

```
The [!DNL Upgrade Compatibility Tool] uses rules defined within our [Coding Standards](https://devdocs.magento.com/guides/v2.4/coding-standards/bk-coding-standards.html), to ensure your project is following Adobe Commerce best practices, and at the same time allowing you to improve and extend the [!DNL Upgrade Compatibility Tool] compatibility analysis by contributing to this project.

Refer to the [Contribute](https://devdocs.magento.com/guides/v2.4/coding-standards/contributing.html)  topic for more information on contributing to this project.
```

File: help/upgrade/upgrade-compatibility-tool/overview.md [hunk 3]
```diff
@@ -19,12 +19,12 @@ Refer to the [Install](../upgrade-compatibility-tool/install.md) for the first s
...
 
-See the [Resources](https://devdocs.magento.com/community/resources/resources.html) page for more information.
+## Resources
+We have developed other resources to help you understand Adobe Commerce upgrades, these are links to key resources:
```
Comment (jeff-matthews) [inline help/upgrade/upgrade-compatibility-tool/overview.md]:
```suggestion

We have developed the following resources to help you understand Adobe Commerce upgrades:
```

File: help/upgrade/upgrade-compatibility-tool/overview.md [hunk 4]
```diff
@@ -19,12 +19,12 @@ Refer to the [Install](../upgrade-compatibility-tool/install.md) for the first s
...
+- [Upgrade guide](https://experienceleague.adobe.com/docs/commerce-operations/assets/adobe-commerce-2-4-upgrade-guide.pdf)
```
Comment (jeff-matthews) [inline help/upgrade/upgrade-compatibility-tool/overview.md]:
```suggestion
- [Upgrade guide](https://experienceleague.adobe.com/docs/commerce-operations/upgrade-guide/overview.html)
```

**Review by danidelcar — APPROVED** [review]
Approved, a couple of comments, thanks!

**Comment by danidelcar** [comment]
@sgonzalezmangana I'll address the changes of the PR

**Review by jeff-matthews — CHANGES_REQUESTED** [review]
The `[!DNL ]` syntax prevents the localization engine from localizing the term. See the [authoring guide ](https://experienceleague.adobe.com/docs/authoring-guide-exl/using/authoring/localization/localize.html?lang=en) for more info.

File: help/upgrade/upgrade-compatibility-tool/overview.md (hunk 2 above)
Comment (danidelcar) [reply help/upgrade/upgrade-compatibility-tool/overview.md]: @jeff-matthews should be  [!DNL Upgrade Compatibility Tool] right?

Comment (jeff-matthews) [reply help/upgrade/upgrade-compatibility-tool/overview.md]: @danidelcar, yes. My mistake. I'll edit my suggestion.

Comment (jeff-matthews) [reply help/upgrade/upgrade-compatibility-tool/overview.md]: @danidelcar, suggestion updated.

**Review by jeff-matthews — APPROVED** [review]

**Review by danidelcar — APPROVED** [review]
good to go after signing Adobe CLA!

**Comment by jeff-matthews** [comment]
@sgonzalezmangana, in the future, please complete the PR description.

**Comment by sgonzalezmangana** [comment]
> 

@jeff-matthews how can I sign the Adobe CLA?

**Comment by danidelcar** [comment]
> > 
> 
> @jeff-matthews how can I sign the Adobe CLA?

@jeff-matthews shared link to sign it with @sgonzalezmangana

**Comment by sgonzalezmangana** [comment]
> > > 
> > 
> > 
> > @jeff-matthews how can I sign the Adobe CLA?
> 
> @jeff-matthews shared link to sign it with @sgonzalezmangana

Signed!

**Comment by sgonzalezmangana** [comment]
Testing notifications @sgonzalezmangana

**Pull request merged**